dask_cuda/benchmarks/utils.py

```diff
@@ -186,6 +191,27 @@ def setup_memory_pool(pool_size=None, disable_pool=False):
     import rmm
 
     rmm.reinitialize(
-        pool_allocator=not disable_pool, devices=0, initial_pool_size=pool_size,
+        pool_allocator=not disable_pool,
+        devices=0,
+        initial_pool_size=pool_size,
     )
     cupy.cuda.set_allocator(rmm.rmm_cupy_allocator)
+
+
+def plot_benchmark(t_runs):
+    import pandas as pd
+    import seaborn as sns
+    import matplotlib.pyplot as plt
```

Comment:
Maybe not totally necessary, but should we error out if the users requests plotting and doesn't have libraries installed, rather than just failing at the end?

Reply:
yeah, i'll add that in

Reply:
Resolved in https://github.com/rapidsai/dask-cuda/pull/423/commits/695eafaaa60adf7e85fae104ac5f0c785b06fd8b